features/page-transition-events.yml

```
@@ -0,0 +1,12 @@
+name: Page Transition Events
+description: "The page transition events `pageshow` and `pagehide` are fired when a document is being loaded or unloaded due to a navigation, such as clicking on a link on a page, or the back button in a browser."
```

consistency, brevity nits

```suggestion
description: "The `pageshow` and `pagehide` transition events fire when a document loads or unloads due to a navigation, such as clicking on a link on a page or the back button in a browser."
```